### PR TITLE
Wrapping big objects and welding lockers now takes time

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
Makes wrapping a locker and crates take time. No more locker catching cheese.
Guy asked for this and it made sense:
https://nanotrasen.se/forum/topic/15602-wrapping-lockers/

Makes welding lockers take time as well.

Both take 1.5 seconds to do.

Code of the wrapping is a mess but that's not what this PR is about.

**Changelog:**
:cl:
tweak: Wrapping lockers and crates now takes 1.5 seconds to do
tweak: Welding lockers now takes 1.5 seconds to do
/:cl:

### CODE REVIEW

**Comment by datlo** [comment]
This should probably also apply to welding lockers which IIRC is instant

**Comment by farie82** [comment]
> This should probably also apply to welding lockers which IIRC is instant

Wasn't sure if that got fixed yet or not. I'll include it in this PR as well

File: code/modules/recycling/sortingmachinery.dm [hunk 1]
```diff
@@ -177,7 +177,9 @@
 		var/obj/structure/closet/crate/O = target
 		if(O.opened)
 			return
-		if(use(3))
+		if(amount >= 3 && do_after_once(usr, 15, target = target))
```
Comment (Fox-McCloud) [inline code/modules/recycling/sortingmachinery.dm]: You know what I did to the last person who cast `usr` when a proc can explicitly pass `user`.

💣 💥

Comment (KasparoVy) [reply code/modules/recycling/sortingmachinery.dm]: Max capped their ass off the codebase

Comment (farie82) [reply code/modules/recycling/sortingmachinery.dm]: hawt

Comment (Citinited) [reply code/modules/recycling/sortingmachinery.dm]: TBF, relative to the rest of this proc, that's just a small drop in the ocean :smile:

**Comment by indijosh** [comment]
Thanks for this!

**Comment by Jazz23** [comment]
literally this is my #1 goto method for capturing people... damn.

**Comment by taukausanake** [comment]
> literally this is my (#)1

That poor, _**poor**_ PR. It thinks we remember it, but no. Just misguided calls to itself. I mean have you seen the number of references it has? What else is weird is all the merge push requests it gets throughout the years

**Pull request merged**